kbase-extension/static/kbase/config/feature-config.json

```diff
@@ -1,5 +1,5 @@
 {
-    "stagingDataViewer": false,
+    "stagingDataViewer": true,
```

Comment:
Oops. Thanks for catching that.
It should probably be turned off until we're ready to start supporting it more (hopefully later this week).